apps/ftu/style/style.css

```diff
@@ -368,12 +368,14 @@ gaia-buttons.forward-only .icon-previous {
 #wifi [data-type="list"] li > p {
   pointer-events: none;
   -moz-padding-end: 6.2rem;
+  line-height: 1.33;
```

Comment:
nah, I always prefer to set it to avoid hierarchy side effects, but no need to change anything, I was just wondering about the reason :)